src/main/java/org/thoughtslive/jenkins/plugins/jira/JiraStepsConfig.java

```diff
@@ -23,14 +23,14 @@
  * @author Naresh Rayapati
  */
 @SuppressFBWarnings
-public class Config extends AbstractDescribableImpl<Config> {
+public class JiraStepsConfig extends AbstractDescribableImpl<JiraStepsConfig> {
```

Comment:
Can this not be renamed to maintain the passivity please?

Reply:
nvm, missed the point in the description, can you also bump the version in pom and change config to point that this is a non passive change, at least this is going to break if any those startup scripts.

Reply:
Update it to 1.5.0 and update the changelog

Reply:
Hi! I made another commit with version and changelog. But I am not sure how to do this: 

> change config to point that this is a non passive change

Reply:
Please add the following plugin to build section: 

```xml
          <plugin>
            <groupId>org.jenkins-ci.tools</groupId>
            <artifactId>maven-hpi-plugin</artifactId>
            <configuration>
              <compatibleSinceVersion>1.5.0</compatibleSinceVersion>
            </configuration>
          </plugin>
```